docs/guides/admin/docs/configuration/multi.tenancy.md

```diff
@@ -67,3 +78,9 @@ used to authenticate users. The file follows the standard ways on configuring Sp
 anything that can go into a Spring Security configuration.
 
 The easiest way of creating that file is probably to create a copy of the already existing `mh_default_org.xml`.
+
+### Other Configuration
+
+Two additional files should be copied: `org.opencastproject.ui.metadata.CatalogUIAdapterFactory-episode-common.cfg` should be copied to `org.opencastproject.ui.metadata.CatalogUIAdapterFactory-episode-common.cfg-tenant1`, and `org.opencastproject.ui.metadata.CatalogUIAdapterFactory-series-common.cfg` should be copied to `org.opencastproject.ui.metadata.CatalogUIAdapterFactory-series-common-tenant1.cfg`.
+
+In each of the new configuration files, change `organization` key to match the tenant id.  Create a copy of the files for each tenant.
```

Comment:
Additionally, it must be ensured that in only one of the ...metadata.CatalogUIAdapterFactory-series-common-....cfg and ...metadata.CatalogUIAdapterFactory-episode-common-....cfg the setting "common-metadata" is set to true. In all others this setting has to be false, otherwise metadata will be available in one tenant only and you get the following errors
Updating property common-metadata of configuration org.opencastproject.ui.metadata.CatalogUIAdapterFactory.75ab8eb8-f0a1-4820-83bd-c28a88fd3288 caused a problem: Only one common metadata catalog adapter is allowed for the type 'events'
org.osgi.service.cm.ConfigurationException: common-metadata : Only one common metadata catalog adapter is allowed for the type 'events' / 'series'